frontend/src/app/components/connections-list/own-connections/own-connections.component.css

```
@@ -167,6 +197,8 @@
 @media (width <= 600px) {
 	.dbGrid {
 		width: 100%;
+		grid-template-columns: 1fr;
+		grid-gap: 8px;
```

Stylelint (17.11.1)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)